examples/app/empty.rs

```diff
@@ -1,3 +1,5 @@
+//! An empty application (does nothing)
```

Comment:
I don't love that this example exists, but that's not this PR's fault.

Reply:
You want me to delete it? 
Not sure if thats something that should be done in this context?

Reply:
I was trying to say that this should be done in a separate PR, yeah.